tools/export_onnx.py

```diff
@@ -0,0 +1,131 @@
+#coding:utf-8
+
+import argparse
+import os
+import sys
+sys.path.append("../")
```

Comment:
Hi @runrunrun1994 , We recommend using pip directly for package management, by using `pip install -e .` in the root directory to avoid using `sys.path.append` you can achieve the same functionality.